### DISCUSSION TITLE
Changes not saved

### DISCUSSION THREAD

**Discussion by PavanLuca** [community/community#62414] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

2 hours ago I made some translations to a file and after committing changes it didn't save them apparently, when I open the file I see the changes I made previously but on the preview I see the file like if I made no changes at all. I'm not getting if the changes were really saved or not, the file preview confuses me.
![Schermata da 2023-08-01 22-12-46](https://github.com/community/community/assets/123581658/e8528c63-c758-4fa9-a32d-378d8afc0514)
![Schermata da 2023-08-01 22-14-00](https://github.com/community/community/assets/123581658/d20f6c23-bcf6-4441-b876-1d72756dc2be)
![Schermata da 2023-08-01 22-15-08](https://github.com/community/community/assets/123581658/0d7a5d1b-8815-4d82-ae5e-886ca117193c)

**Comment by youradrien** [comment]
nice

**Reply by PavanLuca** [reply]
Thanks 😁. It's right due to this kind of stuff that I prefer staying offline when doing certain things. In fact now I won't ever edit a file online on GitHub knowing that's potentially not saved, I just translate my own copy of the original file and upload it once finished or after certain points of translations so I'll never really lose my progresses 😎.

**Comment by PavanLuca** [comment]
Again, I delete a file but for GitHub I did nothing.
![Schermata da 2023-09-13 20-32-43](https://github.com/community/community/assets/123581658/7d7c41b4-c28f-4b10-89b1-093a2d810192)
![Schermata da 2023-09-13 20-33-04](https://github.com/community/community/assets/123581658/c629f6d9-a85d-49a1-bc24-ed4d31df1c34)
Now I won't ever edit anything directly on GitHub, I'll do everything on my laptop until this thing is really fixed and there are proofs that it effectively is, working on internet is overrated.
